src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java

```diff
@@ -344,9 +347,11 @@ public String getSpecifiedPipelinePage(final Model model, Principal principal, L
 			model.addAttribute("addRefProjects", addRefList);
 			model.addAttribute("projects", projectList);
 			model.addAttribute("canUpdateSamples", canUpdateAllSamples);
-                        model.addAttribute("workflowName", workflowName);
+			model.addAttribute("workflowName", workflowName);
 			model.addAttribute("dynamicSourceRequired", description.requiresDynamicSource());
-			model.addAttribute("analysisType", flow.getWorkflowDescription().getAnalysisType());
+			model.addAttribute("analysisType", flow.getWorkflowDescription()
+					.getAnalysisType());
+			model.addAttribute("emailConfigured", emailController.isMailConfigured());
```

Comment:
This is all looking great.  Last change I can see is that if `isMailConfigured` is `false`, the error message in the logs is `E-mail host is not configured, unable to send password reset e-mails.`

Can you make a change in `WebEmailConfig` to update that message to remove the "password reset" part?  That likely should have been done in the past but we can do it here now.  Thanks.

Reply:
Fixed in 63c08683e9697673e7e99401101bfff14c658a05